systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -414,7 +426,7 @@ def fw_router(self):
         self.fw.append(['', '', '-A NETWORK_STATS -i eth2 -o eth0'])
         self.fw.append(['', '', '-A NETWORK_STATS -o eth2 ! -i eth0 -p tcp'])
         self.fw.append(['', '', '-A NETWORK_STATS -i eth2 ! -o eth0 -p tcp'])
-
+        
```

Comment:
3 days of work to find the cause of the bugs and the thing goes with trailing spaces... crap.

Will remove it once I add a marvin test.